src/app/business/identity/userList.component.ts

```diff
@@ -111,9 +112,25 @@ export class UserListComponent implements OnInit, AfterViewChecked {
         return null;
     }
 
-    regConfirmPassword (param: any): ValidatorFn{
+    regConfirmPassword (controlName: string): ValidatorFn{
         return (c: AbstractControl): {[key:string]: boolean} | null => {
```

Comment:
**_nit_**: Make the code more readable. Use variables and aliases that are meaningful. Someone reading the code later should be able to follow it easily.